src/compiler/types.ts

```diff
@@ -861,6 +876,49 @@ module ts {
         expression?: Expression;
     }
 
+    export interface ES6StyleImportDeclaration extends Declaration, ModuleElement {
+        importClause?: ImportClause;
+        moduleSpecifier: StringLiteralExpression;
+    }
+
+    export interface Binding extends Node {
+        name: Identifier;
+        identifierName?: Identifier;
+    }
+
+    export interface ImportClause extends Node {
+        defaultBinding?: Binding;
+    }
+
+    export interface ImportedDefaultBinding extends ImportClause, Binding {

```

Comment:
i do not think you can rename default.. can you?

my understanding is that the grammar is 
```
ImportClause :
    ImportedDefaultBinding 
    ImportedDefaultBinding , NameSpaceImport 
    ImportedDefaultBinding , NamedImports

ImportedBinding :
    BindingIdentifier
```
so you can not have ` import default as blah` as that would be a namespace import, you can only have something like `import default, namespace as blah`